packages/frontend/src/pages/Support/Support.tsx

```diff
@@ -0,0 +1,26 @@
+import { Link } from "react-router-dom"
+
+const Support = () => {
+  return (
+    <div className="legal-text">
```

Comment:
Can you add english support to this like it is done with the other strings using the language lib?